### PR TITLE
fix: astra-assistants types

### PR DESCRIPTION
**Pull request by phact**
Update to use Components with inputs and outputs for compatibility with Text Output and Chat Output components.

Apologies this didn't make it into the last PR.

I also add support for MultilineSecretInput for the dotenv component.

### CODE REVIEW

File: src/frontend/src/components/textAreaComponent/index.tsx
Comment (anovazzi1) [inline src/frontend/src/components/textAreaComponent/index.tsx:1]: broken field:
please update:
- fix the ui in the node
- update the field in the node modal
- update the field in the tweaks
image
image
image

File: src/backend/base/langflow/components/astra_assistants/list_assistants.py
Comment (anovazzi1) [inline src/backend/base/langflow/components/astra_assistants/list_assistants.py:1]: fix broken component:

image

**Review by anovazzi1 — CHANGES_REQUESTED** [review]
Thanks for the pull request. I've noticed some minor bugs in the code. Could you please review and fix them?

File: src/backend/base/langflow/components/astra_assistants/list_assistants.py
Comment (ogabrielluiz) [reply src/backend/base/langflow/components/astra_assistants/list_assistants.py:1]: This is correct.

File: src/frontend/src/components/textAreaComponent/index.tsx
Comment (phact) [reply src/frontend/src/components/textAreaComponent/index.tsx:1]: Fixed!

**Review by anovazzi1 — APPROVED** [review]
LGTM

**Review by anovazzi1 — APPROVED** [review]
LGTM

**Pull request merged**